src/sage/plot/plot.py

```diff
@@ -419,7 +420,8 @@ def y(x): return x*sin(x**2)
 
     sage: g1 = plot(sin(x), 0, 2*pi)
     sage: g2 = plot(cos(x), 0, 2*pi, linestyle="--")
-    sage: (g1+g2).show(ticks=pi/6, tick_formatter=pi)  # long time # show their sum, nicely formatted
+    sage: (g1 + g2).show(ticks=pi/6,        # show their sum, nicely formatted  # long time
+....:                    tick_formatter=pi)
```

Comment:
The bot reports this alignement error. 4 spaces missing before `....:`

Reply:
Thanks, fixed in e5b62efa85